build/images/base/build.sh

```diff
@@ -138,7 +152,8 @@ fi
 function docker_build_and_push() {
     local image="$1"
     local dockerfile="$2"
-    local build_args="--build-arg CNI_BINARIES_VERSION=$CNI_BINARIES_VERSION --build-arg SURICATA_VERSION=$SURICATA_VERSION --build-arg BUILD_TAG=$BUILD_TAG"
+    local custom_build_tag="$CUSTOM_BUILD_TAG"
```

Comment:
we require two different variables, BUILD_TAG & CUSTOM_BUILD_TAG, because when we are pullikg the base images such as "antrea/openvswitch" and "antrea/openvswith-ubi", and other base images we need to specify our build tag, because they are pulled form docker and we do not have the images with custom tags pushed there.

Reply:
but you also modified `build/images/ovs/build.sh` to use your custom build tag, and so `antrea/openvswitch` will be built first with you custom build tag.

Also note that when calling `./hack/build-antrea-linux-all.sh` with `--pull`, the `--pull` flag will *not* be propagated to individual build scripts like this one.

You should use one build tag consistently, and I think the only exception may be the tag for the cache image.

So what I would recommend:
- use `BUILD_TAG` consistently everywhere for the build tag
- define `BUILD_CACHE_TAG` and use it as the tag for the build image
- `BUILD_TAG` defaults to `$(../build-tag.sh)` but can be overridden with `--build-tag`
- `BUILD_CACHE_TAG` is always set to `$(../build-tag.sh)`

Reply:
okay, got it.